File: Oficinas17/while.cpp
// Clube de Programação UPF 2017 //
//   Laços de Repetição: While   //
#include<iostream>
using namespace std;

int  main(){
	int cnt;
	cin >> cnt;
	while(cnt > 0){
		cout << "Autodestruicao em " << cnt << "..." << endl;
		cnt--; // cnt = cnt - 1;
	}
	cout << "BOOM!" << endl;
	return 0;
}

File: Oficinas17/while.java
import java.util.Scanner;

class Main{
	public static void main(String args[]){
		Scanner sc = new Scanner(System.in);
		int cnt;
		cnt = sc.nextInt();
		while(cnt > 0){
			System.out.println("Autodestruicao em " + cnt + "...");
			cnt--; // cnt = cnt - 1;
		}
		System.out.println("BOOM!");
	}
}